vhdbuilder/packer/install-dependencies.sh

```diff
@@ -450,15 +450,13 @@ done
 # Please do not use the .1 suffix, because that's only for the base image patches
 
 AMD64_ONLY_KUBE_BINARY_VERSIONS="
```

Comment:
Leaving this var for now in case we end up adding previous versions again. Should be able to remove it in the near future.